web/src/routes/PoisPage.tsx

```diff
@@ -49,8 +49,6 @@ const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: Ci
   const formatter = useContext(DateFormatterContext)
   const { viewportSmall } = useWindowDimensions()
   const navigate = useNavigate()
-  // eslint-disable-next-line no-console
-  console.log('To use geolocation in a development build you have to start the dev server with\n "yarn start --https"')
```

Comment:
why did you rm this hint?

Reply:
Will readd it if the build is developer friendly